### PR TITLE
add answers

### PR DESCRIPTION
**Pull request by kpie89**

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ In your own words, define what the responsibilities of the model layer are in
 Rails.
 
 ```md
-<!-- your answer here -->
+The model layer is the hard working number cruncher in Rails that nobody sees. The controller layer sends data to the model layer and the model layer takes the request, processes it in the back end and spits out a response to the request back at the controller.
```
Comment (jrhorn424) [inline study.md:34]: The model holds any logic and communicates with the database. Your last sentence really describes what Rails does as a whole, not what the model layer does.

File: study.md [hunk 2]
```diff
@@ -57,5 +57,5 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+The client makes a GET request from the browser. The web server chooses the correct route for that request and sends the information to the controller layer. The controller layer takes that info and makes it so the model layer can read it and process it. The model layer controls the request in the back end and if all goes well, spits the confirmed info back into the controller layer which processes and sends the appropriate response to the server and browser.
```
Comment (jrhorn424) [inline study.md:60]: Not every request is a GET request.

**Comment by jrhorn424** [comment]
:+1:

File: study.md (hunk 2 above)
Comment (kpie89) [inline study.md:60]: Maybe you’re just clarifying so I understand, but the question specifically asks for the process when a client makes a GET request.

**Pull request closed without merging**